src/test/java/core/basesyntax/service/parser/CsvParserImplTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.parser;
+
+import static core.basesyntax.TestObjects.APPLE;
+import static core.basesyntax.TestObjects.FRUIT_QUANTITY;
+import static core.basesyntax.TestObjects.THIRD_LINE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class CsvParserImplTest {
+    private final CsvParser csvParser = new CsvParserImpl();
```

Comment:
Better to create in beforeEach (check other classes as well)

Reply:
Yeah, I did it with classes where we have more than one test